libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -211,6 +229,77 @@ void HarmonicNotchFilter<T>::expand_filter_count(uint16_t total_notches)
     delete[] _old_filters;
 }
 
+/*
+  set the center frequency of a single notch
+ */
+template <class T>
+void HarmonicNotchFilter<T>::set_center_frequency(uint16_t idx, float notch_center, float spread_mul, uint8_t harmonic_mul)
+{
+    const float nyquist_limit = _sample_freq_hz * 0.48f;
+    auto &notch = _filters[idx];
+
+    // scale the notch with the harmonic multiplier
+    notch_center *= harmonic_mul;
+
+    /* disable the filter if its center frequency is above the nyquist
+       frequency
+
+       NOTE: should this be notch_center*spread_mul ? As it is here we
+       do sometimes run the upper notch in a double or triple notch at
+       higher than the nyquist.
+    */
+    if (notch_center >= nyquist_limit) {
+        notch.disable();
+        return;
+    }
+
+    // the minimum frequency for a harmonic is the base minimum
+    // frequency scaled for the harmonic multiplier, the spread
+    // multiplier and the FM_RAT set by the user
+    float harmonic_min_freq = _minimum_freq * spread_mul;
+
+    // we can adjust the attenuation at low frequencies
+    float A = _A;
+
+    // on some vehicles we want to treat zero or very low frequency as
+    // the minimum frequency, on others we want to disable the
+    // notch for low frequencies
+    const bool treat_low_freq_as_min = params->hasOption(HarmonicNotchFilterParams::Options::TreatLowAsMin);
```

Comment:
the option name is not congruent with the docs, would be good to zero in on a term we all agree on and use that.

Reply:
happy to get a suggested variable name, this was chosen to match the option enum name

Reply:
that would be an incorrect description, leaving as TreatLowAsMin